src/functional/$ProtobufReader.ts

```diff
@@ -189,7 +189,7 @@ export class $ProtobufReader {
   }
 }
 
-const utf8 = new TextDecoder();
-const N01 = BigInt(0x01);
-const N7F = BigInt(0x7f);
-const N80 = BigInt(0x80);
+const utf8 = /** @__PURE__ */ new TextDecoder();
+const N01 = /** @__PURE__ */ BigInt(0x01);
```

Comment:
I think these could be change to not execute functions in the root scope.

```js
const N01 = 0x01n;
const N7F = 0x7fn;
const N80 = 0x80n;
```

Reply:
The syntax for `BigInt` without using `BigInt(value)` is to add an `n` in the end of a integer.